cabal-testsuite/PackageTests/LinkerOptions/NonignoredConfigs/cabal.out

```diff
@@ -10,6 +10,13 @@ Building library for basic-0.1...
 Installing library in <PATH>
 # cabal v2-install
 Wrote tarball sdist to <ROOT>/cabal.dist/work/./basic/../dist/sdist/basic-0.1.tar.gz
+Error: [Cabal-7145]
+Packages requested to install already exist in environment file at <ROOT>/cabal.dist/basic0.env. Overwriting them may break other packages. Use --force-reinstalls to proceed anyway. Packages: basic
+# cabal v2-install
```

Comment:
It's extremely confusing the options to `install` are not given here. I guess it's not easy to add the printout of options? Perhaps just a one-off comment that would show in the log?

Reply:
I've shown the install options using `recordHeader`.